middleware/metrics_test.go

```diff
@@ -0,0 +1,63 @@
+package middleware
+
+import (
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+var tR = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "http_requests_total",
+		Help: "Number of get requests.",
+	},
+	[]string{"path"},
+)
+
+var rS = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "response_status",
+		Help: "Status of HTTP response",
+	},
+	[]string{"status"},
+)
+
+var hD = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+	Name: "http_response_time_seconds",
+	Help: "Duration of HTTP requests.",
+}, []string{"path"})
+
+func TestPrometheusRegisters(t *testing.T) {
```

Comment:
This is a super helpful test! Can we have a test for the middleware as well?

Reply:
Yeah, for sure! Let me try to whip something up later today.

Reply:
Okay, got one running. :)